xbmc/video/VideoDatabase.cpp

```diff
@@ -9293,3 +9324,31 @@ bool CVideoDatabase::GetFilter(CDbUrl &videoUrl, Filter &filter, SortDescription
 
   return true;
 }
+
+bool CVideoDatabase::SetVideoUserRating(const int& dbId, const int& rating, const MediaType& mediaType)
+{
+  try
+  {
+    if (NULL == m_pDB.get()) return false;
+    if (NULL == m_pDS.get()) return false;
+
+    if (mediaType == MediaTypeNone)
+      return false;
+
+    std::string sql;
+    if (mediaType == MediaTypeMovie)
+      sql = PrepareSQL("UPDATE movie SET userrating=%i WHERE idMovie = %i", rating, dbId);
+    else if (mediaType == MediaTypeEpisode)
```

Comment:
Was under the impression the Increase/Decrease Rating is only there when you watch something. Not sure if that is the case, anyway I added the tvshow code.